instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const SearchBar = () => {
+  return (
+    <div className="search-bar-wrapper">
+      <div className="image-wrapper">
+        
+      </div>
+      <div>
+        <input type="text" placeholder="Search" />
+      </div>
+      <div className="social-wrapper">
+        <div className="social">
```

Comment:
Great use of font awesome for the social icons definitely keep up the awesome work